py-polars/tests/test_series.py

```diff
@@ -320,9 +320,9 @@ def test_empty():
     a = pl.Series(dtype=pl.Int8)
     assert a.dtype == pl.Int8
     a = pl.Series()
-    assert a.dtype == pl.Float32
+    assert a.dtype == pl.Float64
```

Comment:
I think its better to let an empty series default to f32. When used in an arithmetic operation with another Series (that has values) it may lead to cast to a type makes the operation possible for both. An f64 may force a Series of type f32 to be upcast, doubling its memory usage.

Reply:
All right, I'll see how I can fix that.

Reply:
Fixed.